coap/api/transport.go

```diff
@@ -237,6 +238,10 @@ func receive(svc coap.Service, msg *gocoap.Message) *gocoap.Message {
 		ContentType: ct,
 		Protocol:    protocol,
 		Payload:     msg.Payload,
+		Created: &timestamp.Timestamp{
```

Comment:
I think you should use a function for this: https://github.com/golang/protobuf/blob/84668698ea25b64748563aa20726db66a6b8d299/ptypes/timestamp.go#L113

Reply:
Something like `timestamp.TimestampProto(time.Now())`